src/Bundle/Maker/MakeFactory.php

```diff
@@ -202,6 +203,35 @@ private function defaultPropertiesFor(string $class, bool $allFields): iterable
         $metadata = $em->getClassMetadata($class);
         $ids = $metadata->getIdentifierFieldNames();
 
+        // If Factory exist for related entities populate too with auto defaults
+        $relatedEntities = $metadata->associationMappings;
+        foreach ($relatedEntities as $item) {
+            // if joinColumns is not written entity is default nullable ($nullable = true;)
+            // @see vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/JoinColumn.php LINE 28
+            if (!\array_key_exists('joinColumns', $item)) {
+                continue;
+            }
+
+            if (!\array_key_exists('nullable', $item['joinColumns'][0])) {
+                continue;
+            }
+
+            if (true === $item['joinColumns'][0]['nullable']) {
+                continue;
+            }
+
+            $fieldName = $item['fieldName'];
+
+            $targetEntity = \explode('\\', $item['targetEntity']);
+            $targetEntity = \end($targetEntity);
+            $factory = \ucfirst($targetEntity).'Factory';
+
+            if ($this->hasFactory($factory)) {
+                yield \lcfirst($fieldName) => \ucfirst($factory).'::new(),';
```

Comment:
Let's yield the Factory's FQCN if not in the same namespace as this new factory.